### PR TITLE
Export secondary entrypoints for each Angular component wrapper

### LINKED ISSUE #1447: Compilation failed for Angular apps using @sl-design-system/angular > 0.2.0
### Provide a general summary of the issue here

When updating "@sl-design-system/angular" to "1.0.1", building of your application does not work anymore. This is due to the fact that @sl-design-system/angular comes in 2 flavors: fesm and esm builds.
In the flattened esm build (fesm) all the imports are grouped together, so also the imports from the wrapper components. 

We do not have all slds components installed, so these give compilation errors.

image

image

### 🤔 Expected Behavior?

Build succeeds, all imports can be resolved

### 😯 Current Behavior

see general summary, including screenshots

### 💁 Possible Solution

install all slds components, fix the imports of the wrapper components, or ... ?

### 🖥️ Steps to Reproduce

Easierst way to reproduce is to update  repo [angular-demo](https://github.com/sl-design-system/angular-demo) to "@sl-design-system/angular" to "1.0.1"

Or use my branch on Max-Online [VO-12102](https://bitbucket.org/sl-technology/vooruit-frontend/branch/VO-12102)

### What browsers are you seeing the problem on? (only for bugs in code)

Chrome

### If other, please specify.

_No response_

### What operating system are you using?

MacOs

### 🧢 Your product/team

Max Online

### 🎨 Your Theme(s)

Max Online
```[tasklist]
### Tasks
- [ ] update demo app to reproduce the issue
```

### PR DESCRIPTION
**Pull request by jpzwarte**
Fixes #1447

### CODE REVIEW

File: scripts/cem-plugin-angular-wrapper.js [hunk 1]
```diff
@@ -35,10 +35,18 @@ ${events.length ? `\n${events.map(event => event.code).join('\n')}\n` : ''}}
...
 
-const generateIndex = async (components, outDir) => {
-  const indexSrc = components.map(component => `export * from './${dasherize(component.name)}.component';`).join('\n');
+const generatePublicApis = async (packages, outDir) => {
+  for (const [packageName, files] of packages) {
+    await writeFile(join(outDir, packageName, 'public-api.ts'), files.map(f => `export * from './${f}';`).join('\n').concat('\n'), 'utf8');
 
-  await writeFile(join(outDir, 'index.ts'), indexSrc, 'utf8');
+    await writeFile(join(outDir, packageName, 'ng-package.json'), `{
+  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
```
Comment (elgervb) [inline scripts/cem-plugin-angular-wrapper.js:43]: small thing maybe. but as of `ng-packagr` `17.3.0`, you should use schema `ng-packagr/ng-entrypoint.schema.json` for secondary entrypoints. 
But then again.. if this works..

File: packages/angular/stories/forms.stories.ts [hunk 2]
```diff
@@ -9,31 +9,27 @@ import {
...
+import { FormComponent } from '../src/form/form.component';
+import { CheckboxGroupDirective } from '../src/forms/checkbox-group.directive';
+import { CheckboxDirective } from '../src/forms/checkbox.directive';
+import { RadioGroupDirective } from '../src/forms/radio-group.directive';
+import { SelectDirective } from '../src/forms/select.directive';
+import { SwitchDirective } from '../src/forms/switch.directive';
+import { TextAreaDirective } from '../src/forms/text-area.directive';
+import { TextFieldDirective } from '../src/forms/text-field.directive';
+import { InlineMessageComponent } from '../src/inline-message/inline-message.component';
+import { RadioGroupComponent } from '../src/radio-group/radio-group.component';
```
Comment (anna-lach) [inline packages/angular/stories/forms.stories.ts:26]: Maybe we can also add wrappers for newer components like `progress bar`?

**Review by Diaan — APPROVED** [review]

File: packages/angular/stories/forms.stories.ts (hunk 2 above)
Comment (Diaan) [reply packages/angular/stories/forms.stories.ts:26]: The progress bar wrapper exists, it just isn't in this example story.

**Comment by Diaan** [comment]
When reviewing please also check https://github.com/sl-design-system/angular-demo/pull/6

**Pull request merged**